doc/overview/people.rst

```diff
@@ -27,6 +27,7 @@ Steering Council
 * `Guillaume Favelier`_
 * `Luke Bloy`_
 * `Mainak Jas`_
+* `Mathieu Scheltienne`_
 * `Marijn van Vliet`_
```

Comment:
just a note that alphabetical order was violated here. Open invitation for someone to fix it when a hefty script is running or when they're stuck on some other problem.

Reply:
fixed in b0f8ce69c7258319624aebdd6de63b23e1c78ad5